apps/api/src/player/player.controller.ts

```
@@ -0,0 +1,68 @@
+                    { [`members.recruit.${response.data.uuid}`]: { $exists: true } },
+                ],
+            });
+
+            response.data.guild = null;
+            // Return if the player is not in any guild, no need to add guild data
+            if (!guild) return response.data;
+
+            // Determine the player's rank in the guild
+            let playerRank = '';
+            for (const rank of ['owner', 'chief', 'strategist', 'captain', 'recruiter', 'recruit']) {
+                if (guild.members[rank] && guild.members[rank][response.data.uuid]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)